examples/exex/op-proposer/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use alloy_network::EthereumSigner;
+use alloy_provider::ProviderBuilder;
+use alloy_signer_wallet::LocalWallet;
+use config::OpProposerConfig;
+use db::L2OutputDb;
+use futures::Future;
+use op_proposer::OpProposer;
+use reth_exex::ExExContext;
+use reth_node_api::FullNodeComponents;
+use reth_node_ethereum::EthereumNode;
+use rusqlite::Connection;
+use std::path::Path;
+
+pub mod config;
+pub mod db;
+pub mod op_proposer;
+pub mod tx_manager;
+
+async fn init_exex<Node: FullNodeComponents>(
+    ctx: ExExContext<Node>,
+) -> eyre::Result<impl Future<Output = eyre::Result<()>>> {
+    // TODO: pass the config path as an arg instead of hardcoding
+    let config_file = Path::new("./op_proposer.toml");
```

Comment:
I think we'd need to join this with the MANifest env var to always get the right path